ctf/tests/core_spec.lua

```diff
@@ -0,0 +1,112 @@
+function assert_tables_equal(expected, actual)
+    for key,value in pairs(expected) do
+        assert.equal(expected[key], actual[key])
+    end
+end
+
+function assert_keysets_are_equal(sample, actual)
+    for key in pairs(sample) do
+        assert.not_nil(actual[key])
+    end
+    for key in pairs(actual) do
+        assert.not_nil(sample[key])
+    end
+end
+
+
+describe("tests orderPairs", function()
+    setup(function()
+        require("ctf/tests/mock/minetest")
```

Comment:
You can add two path entries. One is expected relative from mod root and second is relative from project root.
Lua will check every path entry until it finds required file.